scripts/run-django-dev.sh

```diff
@@ -3,9 +3,15 @@
 # This script runs the django app
 
 python3 manage.py collectstatic --noinput --clear
+
+# run initial django migrations
 python3 manage.py migrate --noinput
+
+# populate cache table
 python3 manage.py createcachetable
-python3 manage.py migrate --noinput
+
+# run ONLY data migrations
+RUN_DATA_MIGRATIONS=true python3 manage.py migrate --noinput
```

Comment:
Its run this way since the data migrations are intended to run once all the actual schema migrations have run. There is no elegant way to add "latest schema migration x" as a dependency of the first data migration so instead we make sure regular schema migrations are run and then the data migrations go in afterwards.